hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -650,6 +656,12 @@ private FlinkOptions() {
       .defaultValue(30)
       .withDescription("Maximum number of groups to create as part of ClusteringPlan. Increasing groups will increase parallelism, default is 30");
 
+  public static final ConfigOption<Integer> CLUSTERING_TIMEOUT_SECONDS = ConfigOptions
+      .key("clustering.timeout.seconds")
+      .intType()
+      .defaultValue(1200) // default 20 minutes
```

Comment:
The option can be removed because it is never used.

Reply:
fixed.